rttr-ru.po

```diff
@@ -264,8 +270,12 @@ msgid "A catapult is firing upon us!"
 msgstr "Катапульта стреляет по нам!"
 
 #: libs/s25main/gameData/BuildingConsts.cpp:161
-msgid "A gold mine allows you to prospect for valuable gold deposits. For this, it is necessary to ensure that the miners are provided with plenty of food."
-msgstr "Золотая шахта даёт вам доступ к такому ценному ресурсу как золото. Но таких полезных шахтёров надо кормить."
+msgid ""
```

Comment:
Something went wrong here and in other places. I don't think "msgid" can/should be changed